### ISSUE TITLE
Create better visual art for planet descriptions

### ISSUE THREAD

**Issue by Platypants** [endless-sky/endless-sky#3465]
I feel that most of the planet photographs are sadly lacking on the creative side, and don't match the integrity and fun of the sprites, outfits, sounds, and story writing. Seeing the same mountains, 20th century space stations, and recognizable Earth cityscapes repeated ad nauseam takes me out of the game world. On my first playthrough of the current version (0.9.8), before looking at the forums, I thought that most of the images were placeholders for original artwork to come.

I think a call for visual artists to create landscapes would be appropriate. I will probably contribute a few landscape images myself. This relates directly to #790 (why was this closed so quickly?) and indirectly to #3389 (open).

I am new to this community, and to open source gaming in general. It seems to me that github is good for technical concerns but not so good for creative ones. I feel more at home writing poetry than C++ and I find this forum intimidating. Hopefully this is an appropriate place to voice an aesthetic concern.

**Comment by Amazinite** [comment]
>This relates directly to #790 (why was this closed so quickly?)

#790 was closed because #799 was merged.

**Comment by Platypants** [comment]
Thanks @Amazinite. I am going to join the Google Group too.

**Comment by Amazinite** [comment]
So are there any specific landscape images that you could point to that are dissatisfying to you, and would you be able to provide an example of something you consider better than the current images?

**Comment by Alkallid** [comment]
land/hills2 has some snow that looks very painted on.

**Comment by Platypants** [comment]
The most obvious examples for comparison are planet photos from our predecessor EV: Nova. Little details are nice, like a ship flying in front of otherwise generic mountains and shorelines:
![ryll](https://user-images.githubusercontent.com/19715924/34342333-0d0726e0-e97b-11e7-92a5-6560c7b04632.png) 

Surreal lighting and abstract buildings/structures can also go a long way: 
![retsim](https://user-images.githubusercontent.com/19715924/34342365-f43c6548-e97b-11e7-91ff-66ae9491188a.png)

ES has gone one step in this latter direction with the strange colors of the Quarg worlds. However, that still looks like what it is, a single random photo applied to dozens of planets/stations, rather than an intentional piece of in-universe artwork for a single planet/station. My general argument would be that original artwork (even with found photos) is superior to raw found photos. I also believe we can make planet images that are better than EV: Nova's, not least because software imaging has come a long way since 2003.

**Comment by Platypants** [comment]
To be honest, there are only 6 or 7 photos in the whole game that I think are appropriate for the in-game reality.

The Quarg photo I was referencing is this one:
![station2](https://user-images.githubusercontent.com/19715924/34342498-c96a9f2a-e97f-11e7-8386-bf104292eecd.jpg)

Space station graphics are particularly dissatisfying. If it's 3017 in the game, I don't want to see a space shuttle hangar, for instance: 
![space5](https://user-images.githubusercontent.com/19715924/34342512-52b919a0-e980-11e7-8457-19a2c809abf4.jpg)
Even in 2017 the space shuttle is a bit dated.

**Comment by tehhowch** [comment]
This is a known issue, and contributions to the game's art are much more desirable than complaints that the art is not up to a given standard.

Please review the wiki pages regarding help wanted, and artwork:
https://github.com/endless-sky/endless-sky/wiki/HelpWanted
https://github.com/endless-sky/endless-sky/wiki/StyleGoals

I don't do artwork development, so I happily refrain from criticizing the game's existing artwork.

**Comment by Amazinite** [comment]
Just a heads up, v0.9.9 is adding a bunch of futuristic images for space stations: 3d76c34

**Comment by Platypants** [comment]
@tehhowch I agree with you, and I intend to contribute. However, it's hard for me as a newcomer to see what the known issues are. I didn't see this general issue raised elsewhere in this repository.

**Comment by jafdy** [comment]
There's a pile of images that didn't get added to the game when I was running a set of contests for the landscape images https://drive.google.com/open?id=0B0mWx-Fhhhz7WkxLemZNeGFLbUE. Stations are much more the problem now than mountains (I think there are 30 something pictures of mountains.)

**Comment by Pointedstick** [comment]
Adding some ships flying around would be a nice touch. That shouldn't be too hard to do with a little Blender- and GIMP-fu. Please feel free to try your hand at it!

**Comment by endless-sky** [comment]
One issue with putting anything artificial in an image (buildings, ships, etc.) is that the landscape image then becomes usable only for a single species.

It also takes a considerable amount of skill to photoshop a ship into an image in a natural-looking way. You need to match the lighting angle, atmospheric haze, etc. I tried it early on, but wasn't able to achieve a result I was happy with.

**Comment by Pointedstick** [comment]
> One issue with putting anything artificial in an image (buildings, ships, etc.) is that the landscape image then becomes usable only for a single species.

I think we should aspire to never duplicate art, which would make this concern moot.

**Comment by KiLEdEnNis** [comment]
> Adding some ships flying around would be a nice touch. That shouldn't be too hard to do with a little Blender- and GIMP-fu. Please feel free to try your hand at it!

there goes 1 of my mod ideas I'd get to in like 3 years
but I approve; it would make the game feel more "alive", maybe sometime a few major shipyard planets have several human ships in the image, if someone somehow managed to do it properly, the only problem that would be left is that only 1 species can use the image, but ideally we don't want any duplicate landscape image anyway

**Comment by Nechochwen-D** [comment]
Using HDR world images and high quality ship models it should be fairly easy in Blender.

**Comment by MCOfficer** [comment]
if anyone wants to try it, it would be cool to see an the arach ship that has been turned into a house, as mentioned in their description.

also, if you need gimp-fu/python-fu, feel free to ask.

**Comment by rossington** [comment]
A basic HDR workflow using the existing ship models produces results comparable to the EV examples (see attached image). Motion blur is pretty good at hiding what would otherwise be far too simple textures on the model. A good place for HDR images is https://hdrihaven.com (the one used in the example is licensed under CC0, see https://hdrihaven.com/hdri/?c=skies&h=kiara_2_sunrise).
![mountain_with_ships_example](https://user-images.githubusercontent.com/19151265/36647541-12951b00-1a7f-11e8-9bee-d2b7b64a9285.png)

**Comment by nylki** [comment]
@rossington That's some good work, it would fit great in my opinion!

**Comment by MCOfficer** [comment]
this method works surprisingly well. 
However, the question arises how exactly ships take off - do they use some special athmospheric boosters? Is the start more like an airplane or vertically, like a rocket (see [nasa2](https://github.com/endless-sky/endless-sky/blob/master/images/land/nasa2.jpg), [nasa3](https://github.com/endless-sky/endless-sky/blob/master/images/land/nasa3.jpg), [nasa6](https://github.com/endless-sky/endless-sky/blob/master/images/land/nasa3.jpg))?

Also, one should stick to ships that have a clear "front", because the bulk freighter that i used - well, judge for yourself...
![bulk](https://user-images.githubusercontent.com/22377202/36789364-9541c530-1c91-11e8-82c2-4c8acf0577f3.png)

**Comment by jafdy** [comment]
Ships take off using repulser engines that are very powerful but only work in gravitational fields.

**Comment by CADREL** [comment]
Hey all. I recently stumbled upon this pull request and thought I could help out. I took a few days to make the following station for Luna. I had a few ideas I was basing the model on:

1) I think that anything in the Sol system should be the paragon of design and aesthetics as it was the first to be colonised, as well as being the capital. Therefore the trade and economic value would be reflected. i.e. Most capitals around the world are the best in terms of infrastructure and funding.

2) There are a few requirements listed about landscape pictures from the style goals post (https://github.com/endless-sky/endless-sky/wiki/StyleGoals). Some things like not having foreground objects are understandable. But I don't quite understand why people in the picture are not preferred. At least, maybe put them in the far background? - I just think that without people, the picture (as well as other landscape/station pictures) look desolate and lifeless.


There are a few minor things I want to maybe tweak; like add some extra small objects in the store. Let me know what you all think and what I should do to improve and how to fit it better to the ES style. Also, see if anyone can find the joke I put in the picture.

P.S. This is my first post on GitHub so let me know if there is something that I did or wrote incorrectly.

![Render5a](https://user-images.githubusercontent.com/59090730/71520142-929bbd00-28f5-11ea-903c-8f5439ba978d.png)

**Comment by Amazinite** [comment]
Looks amazing! Not sure if I'm keen on the restrictiveness of everything, though. It's so reflective that the ceiling lights are bouncing off of the floor to the ceiling and back to the floor again. 
![image](https://user-images.githubusercontent.com/17688683/71522968-eb873900-2894-11ea-97ec-387610e478aa.png)

And as much as I find the "The Filming of Fast and Furious 387 has been confirmed" blurb hilarious, we tend to stray away from such direct modern cultural references.

**Comment by CADREL** [comment]
@Amazinite Thanks for the feedback. I'll rework the model to your input. 

> I think those are good assumptions, but they aren't really reflected in the story of the game. Sol, being the home system of humanity, actually has some of the oldest infrastructure. Take the descriptions of Earth for example as being a crime ridden, overpopulated, rundown planet that everyone wants to get off of. The areas around Parliament might be high end, but anything beyond that is going to be more like a favela, and I can almost guarantee you that basically everyone in Parliament lives off world.
Mars is another good example of the capital system not being the best. Mars was the subject of the first and therefore worst terraforming experiment. The best terraformed worlds are in the Paradise Worlds region, not in Sol.

Hmm, seams counter-intuitive to economic pressures and government intervention. As an example, Beijing and Shanghai, two cities in China that each hold over 25 million people, are very clean and new infrastructure is being built, while demolishing the old. Indeed the surrounding villages (or sub-city as they call it) is packed full of people in bare minimum infrastructure. Don't get me wrong, the cities are engulfed in people, but they are controlled as they play vital centres for politics and trade (respectively). I think old infrastructure would be replaced with more efficient one if the money is available. Singapore as an example as a process called 'en bloc'. Being a city-state on an limited island, old housing units (from the 1980 and older) are being bought by developers, demolished and then build new ones that can house more people at higher prices, hence turning a profit. I don't think it would be a far stretch to say that if Sol is the economic (and political) centre for the Republic, it would be trying to present itself as the epitome of development – that's why most capitals are the best cities in respective countries.

But you are right, the description doesn't really match the station. Thanks for the examples given on what type of stations I can use as references. I will take some time to rework the station – I can see that this might involve redoing everything, so I think I'll probably start with a new file.

**Comment by jafdy** [comment]
I think Sol has been relegated to being only the political figurehead. Notably the system of thinking is predominantly western, the development in Syndicate space may follow China for the reasons that you have stated but Earth (and sol in general) have only the Republic parliament (and Earth) as points of interest. It is also quite clear that it is hard to move large numbers of people long distances so Earth will stay overpopulated and with few work opportunities there is little profit to be made in housing development. You can start to see why moving to a new planet to start again is a popular idea.

**Comment by CADREL** [comment]
Hey all again. I kept working on a few ideas I got for the landscape photos. I haven't redone the Luna space-station yet but I wanted to try more landscaping in the mean time. I did the landscape for Antipode, based on its planet description. I wanted to know if this kind of style would fit ES before I start creating more landscape renders.

The image below is only 960*540 (50% of 1920*1080) as to speed up the render. If you all think its a good image, I can re-render it at 1920*1080.

I added the spaceport (based on the spaceport's description) to the landscape scene to make to image feel more inclusive of the space-travel feel. Let me know what you all might think about it. If this image is good and you think that this style can really fit ES, I'll start making more models and after a get a good batch (like around 10) I'll make a pull request. That's the procedure right?

Anyway thanks for the feedback!

![Composition2](https://user-images.githubusercontent.com/59090730/72197837-fdee9e80-3460-11ea-868b-22e1042766aa.png)

**Comment by Amazinite** [comment]
Looks like a great start. Part of the planet description mentions though:
>The inhabitants of Antipode are among the best downhill skiers in the galaxy, and they have made some attempts to draw off-world tourists to the ski resorts here, but the weather is so hostile that they have had very little success. 
And there's also this in the spaceport:
>In one corner is a counter advertising skiing adventures, but no one is staffing it; they must be out to lunch or taking the day off.
So perhaps the mountains could have more... "inviting" slopes, and include a ski lift or two. This also doesn't look all that snowy to me, just dark. I'm guessing that the darkness comes from the mention of the world being far from a dim star, but it seems a little overdone to me. (The distance from the star also focuses on the warmth that it brings, not really the brightness of the light.)

**Comment by Zitchas** [comment]
Nice picture! I agree with Amazinite's comments about tweaking it to have some more potential slopes. That being said, there's almost certainly other cold-weather ports that might fit better.

If you happen to get into doing gaseous planets, I'm looking forward to making a lot more of them visitable in the near future.

**Comment by CADREL** [comment]
Thanks for the feedback @Amazinite & @Zitchas. Yeah, that’s why I added the ‘escalator tunnel’ on the right side of the spaceport. It’s how people can exit to the outside. But I think you’re right, I can add some ski lifts.

And yeah, it’s could be a bit dark, I’ll try to tweak the sun brightness and the fog intensity - it’s causes the most darkness.

I’ll try to ‘roundoff’ the mountains. I’ll have to mess with some math nodes as the mountain are procedural.

@Zitchas I don’t know if I’ll be doing gaseous planets :D The render time with any volumetric is almost tripled. And my computer isn’t too great, so even the pre-render takes 10s of minutes. I’ll try some landscaping like for New Boston. The more I can use procedural nodes and particle systems, the “less work” I need to do, the more images I can create.

**Comment by Zitchas** [comment]
That sounds good.

As far as render times go, what software are you using?

**Comment by CADREL** [comment]
Redone the render as you suggested. I agree, this way looks much better. Thanks for your feedback.

@Zitchas Blender 2.8. P.S. this render took 7 hours - it has 2 volumetrics, the fog on the bottom and the thick clouds on the top.

![Composition3](https://user-images.githubusercontent.com/59090730/72212858-977c8580-351f-11ea-8424-86e17baf9abe.png)

**Comment by Amazinite** [comment]
Looking much better. 👍

**Comment by Zitchas** [comment]
Agreed, that looks really good.

If you are on the discord group, send me a message. I can help render stuff now and then. Provided you can give me a "ready to hit 'render'" image. I believe there was some kind of render farm thing setup there too.

**Comment by jafdy** [comment]
Sheepit, if your image can be split it will run really fast over there. As far as I know they are currently out of frames to render.

**Comment by MCOfficer** [comment]
> 
> 
> Sheepit, if your image can be split it will run really fast over there. As far as I know they are currently out of frames to render.

Yes, but it has some caveats: You will have to bake any physics simulations (which i assume the fog is), and you will have to check CPU render only if you want your volumetrics to last - they can introduce visual bugs when rendered on GPU.
https://sheepit-renderfarm.com/

**Comment by CADREL** [comment]
Good suggestions. But I don’t mind letting my computer while I sleep (as long as the render time is under 9 hours :D)
